eng/Versions.props

```diff
@@ -29,7 +29,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependency from https://github.com/dotnet/templating -->
-    <MicrosoftDotNetCommonItemTemplatesPackageVersion>5.0.0-preview.1.20119.2</MicrosoftDotNetCommonItemTemplatesPackageVersion>
+    <MicrosoftDotNetCommonTemplatesPackageVersion>5.0.0-preview.1.20224.2</MicrosoftDotNetCommonTemplatesPackageVersion>
```

Comment:
Will darc find this new property name?

Reply:
And tests are failing due to missing this new package version

Reply:
What is the package version and the package name for latest 
Microsoft.DotNet.Common.ProjectTemplates.5.0 and Microsoft.DotNet.Common.ItemTemplates . I cannot restore it

Reply:
Updated the PR to the more minimal changes - just using the same version property which was already being set by darc for the project templates rather then renaming it.

Reply:
version 5.0.0-preview.1.20124.3 looks to be the current published version with the required changes